demos/browser/app/messagingSession/messagingSession.ts

```diff
@@ -19,6 +19,7 @@ import {
 } from 'amazon-chime-sdk-js';
 
 import { ChimeSDKMessagingClient, GetMessagingSessionEndpointCommand } from '@aws-sdk/client-chime-sdk-messaging';
+import PrefetchOn from "../../../../src/messagingsession/PrefetchOn";
```

Comment:
You should be using this import from `amazon-chime-sdk-js` and not relative from the `src/..`. Could you please check other imports.